### DISCUSSION TITLE
Support of Bootstrap v5 in the Orchid platform

### DISCUSSION THREAD

**Discussion by angularcircle** [orchidsoftware/platform#1605] [General]
Last month Bootstrap `5beta2` was released, according to Bootstraps blog `beta3` - that should be released in near future (guessing next month since backlog isn't long) - will be promoted to v5 stable. 

There is a branch of Orchid dedicated to it - is it ready and just waiting for stable release, was abandoned and there are no plans to add support for it at the moment? Orchid will switch to Tailwind or other lib in near future? In other words: what is current Bootstrap dev status within Orchid context?

**Comment by tabuna** [comment]
Hi @angularcircle, we have a lot of progress on bootstrap and other stuff in the master branch. The bootstrap branch was created by one of the regular contributors but has been abandoned and removed soon.


At the moment, all fields & layouts support bootstrap _5.0.0-beta2_, but we have not yet managed to remove the _jQuery_ code in some places. So far, this is not a stone because I still have some improvements not related to JS/CSS for version 10.x
 
You can check out the master branch yourself. **Not for production**
![image](https://user-images.githubusercontent.com/5102591/110402054-c0e56280-808b-11eb-9ac4-79e8bf0246ff.png)

We have precise plans to add a new version of bootstrap in the next major release.


> Orchid will switch to Tailwind or other lib in near future?

I do not think so. I have no good experience with Tailwind, and updating for the sake of updating is highly questionable.


>  In other words: what is current Bootstrap dev status within Orchid context?

Implemented, but minor flaws are most likely still present (In progress.)

----
I will be glad if you or anyone else will join and help me release the release earlier and better.

**Reply by angularcircle** [reply]
Thanks for quick and extensive answer. At the moment I'm experimenting with laravel cms-like solutions. If Orchid will be chosen (and probably it will be), I'll try to contribute a little bit
